Add dfs flag for `typeclasses eauto` (#14693)
coq/coq: doc/changelog/04-tactics/14693-alizter+typeclasses-eauto-dfs-flag.rst

- **Added:**

>   `typeclasses eauto dfs` syntax which locally enables depth-first search analogously to the `typeclasses eauto bfs` syntax.

Reviewer: ```suggestion
  `dfs` option in :tacn:`typeclasses eauto` to use depth-first search
```

It's convenient for readers to have the hyperlink to the tactic.
